### ISSUE TITLE
Unable to download image classification dataset

### ISSUE THREAD

**Issue by khawar-islam** [eccv22-ood-workshop/ROBIN-dataset#2]
Hello @DTennant 

I am unable to download train set for image classification task. **"Sorry, the file you have requested does not exist."**

**Comment by DTennant** [comment]
Hi,
I just checked this, it seems that the original google account used to upload the dataset is deleted somehow.
I am uploading the dataset again, and I will update the link the README.

**Comment by DTennant** [comment]
Hi the link should be fixed by now

**Comment by khawar-islam** [comment]
Thank you @DTennant

**Comment by DTennant** [comment]
Hi,
Thanks again for your interest in our workshop, and we apologise for the late update.
I have update the README in this repo and update a link to the full test set and labels in the README, the codalab submission server will also be online within this week.

Here are some changes to the competition:

The Phase-1 of the competition will not be a code submission challenge, we have released all the test data and labels in this repo. And Phase-1 will last longer than original planed, we will ask each team to provide a description of their developing environment at the end of Phase-1, Phase-2 will still be code submission challenge.
We will be using Top-1 accuracy for image-classification, mAP@50 for object detection, and Acc@pi/6 for pose estimation as the metric, the IID test performance will also be considered as per request of the sponsor, we will penalize submissions that are significantly different in IID performance with our baseline.
The only limitation now is that the model should only be trained on the given training set and/or the ImageNet-1k dataset, no additional dataset is allowed. You can use any ensemble, data augmentation, or test-time training techniques.
Thanks again for your patience.

Best regards

**Comment by khawar-islam** [comment]
@DTennant Where is the new link of the classification dataset? I think you have updated the classification dataset, please let me now.
